tests/dcps_tests.lst

```diff
@@ -407,7 +408,7 @@ tests/DCPS/Instances/run_test.pl multiple_instance multiple_datawriter nokey: !D
 tests/DCPS/SkipSerialize/run_test.pl rtps_disc: !DCPS_MIN !NO_MCAST RTPS !DDS_NO_OWNERSHIP_PROFILE !OPENDDS_SAFETY_PROFILE
 tests/FACE/Unit/run_test.pl qossettings: !DCPS_MIN !WCHAR !NO_BUILT_IN_TOPICS
 tests/FACE/Unit/run_test.pl memorypool: !DCPS_MIN !WCHAR !NO_BUILT_IN_TOPICS
-tests/FACE/Unit/run_test.pl safetyprofilepool: !DCPS_MIN !WCHAR !NO_BUILT_IN_TOPICS OPENDDS_SAFETY_PROFILE
+tests/unit-tests/dds/DCPS/run_test_safety.pl safetyprofilepool: !DCPS_MIN !WCHAR !NO_BUILT_IN_TOPICS OPENDDS_SAFETY_PROFILE
```

Comment:
This removed the non-migrated FACE unit tests from the test lineup.

Reply:
What do you mean by this? Are you saying the safetyprofilepool test should not be in the dcps test list now, or are you saying I should delete the other FACE tests from the list?

Reply:
You're right, I read that wrong. I thought it was running all the FACE unit tests in one entry.